scripts/runtime-dependency-version-check/runtime-dep-version-check.js

```diff
@@ -108,3 +108,36 @@ if (errors.length) {
   const violations = errors.join(", ");
   throw new Error(violations + " have inconsistent declared versions.");
 }
+
+for (const pkg of nonClientPackages) {
```

Comment:
The generated protocol test packages in `/private` *should* have their dependencies updated the same way as client packages. Should those get handled separately? They shouldn't ever be out of sync with the clients anyways, but this could potentially paper over an issue where they get updated independently of the right codegen changes being applied to those test packages.

Reply:
I've moved the private entries into the client list so they get checked and contribute to the canonical version that gets written to the rest of the packages.